### PR TITLE
feat: move ga events to amplitude

### PR DESCRIPTION
**Pull request by just-toby**
## Description

this is part of #6554 based on the feedback on that PR - instead of completely removing those events we'll log them to amplitude.

in a follow up PR i'll actually delete the dead GA code

## Screen capture

image

image

image

image

image

### CODE REVIEW

**Review by JFrankfurt — APPROVED** [review]

File: src/pages/AddLiquidity/index.tsx [hunk 1]
```diff
@@ -434,11 +433,6 @@ function AddLiquidity() {
     const maxPrice = pricesAtLimit[Bound.UPPER]
     if (maxPrice) searchParams.set('maxPrice', maxPrice.toSignificant(5))
     setSearchParams(searchParams)
-
-    sendEvent({
-      category: 'Liquidity',
-      action: 'Full Range Clicked',
-    })
```
Comment (cbachmeier) [inline src/pages/AddLiquidity/index.tsx:441]: are we no longer tracking this one?

**Review by cbachmeier — COMMENTED** [review]
Looks good, just had a Q about 1 event

File: src/pages/AddLiquidity/index.tsx (hunk 1 above)
Comment (just-toby) [reply src/pages/AddLiquidity/index.tsx:441]: no i don't think it's necessary but we should be including more details in the add events... updating

Comment (just-toby) [reply src/pages/AddLiquidity/index.tsx:441]: also this wasn't on the list of events Blair requested we migrate in #6554

**Review by cbachmeier — APPROVED** [review]

**Pull request merged**